e2e_test/streaming/cross_db_mv.slt

```diff
@@ -0,0 +1,88 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO true;
+
+statement ok
+create database d1;
+
+statement ok
+create database d2;
+
+statement ok
+set database to d1;
+
+statement ok
+create table t1(v1 int);
+
+statement ok
+insert into t1 select * from generate_series(1, 10000);
+
+statement ok
+set database to d2;
+
+# requires log store to support cross-db mv
+statement error
+create materialized view mv1 as select * from d1.public.t1;
+
+statement ok
+set database to d1;
+
+statement ok
+create subscription sub from t1 with(retention = '1D');
+
+statement ok
+set database to d2;
+
+query ok
+select count(*) from d1.public.t1;
+----
+10000
+
+statement ok
+create materialized view mv1 as select * from d1.public.t1;
+
+query ok
+select count(*) from mv1;
+----
+10000
+
+statement ok
+create table t2(v2 int);
+
+statement ok
+insert into t2 select * from generate_series(1, 100);
+
+statement ok
+create materialized view mv2 as select t2.v2 from d1.public.t1 join t2 on t1.v1 = t2.v2;
+
+query ok
+select count(*) from mv2;
+----
+100
+
+statement ok
+set database to d1;
+
+# need to drop cross-db mv first
+statement error
+drop table t1 cascade;
+
+statement ok
+set database to d2;
+
+statement ok
+drop materialized view mv1;
+
+statement ok
+drop materialized view mv2;
+
+statement ok
+set database to d1;
+
+statement ok
+drop table t1 cascade;
+
+statement ok
+drop database d2;
+
+system ok
+risedev psql -c 'drop database d1;'
```

Comment:
Seems not supported to run command in madsim.

Reply:
Yes, I also discovered that using a system command to drop the database is unsafe, as it may disrupt the ongoing SQL connection while it has been set to database `d1`.
Unfortunately, I found that it is impossible to directly run the entire cross db mv logic in slt. I will comment out this test in this PR first and add some shell scripts later to test it.

Reply:
Can we assume that the original database is always `dev`, and then switch back to dev and then drop database d1?

Reply:
Maybe we can use the `connection` statement to perform the operation on other database in a new connection. The statement can be used like the following example.
```
connection other
query T
show RW_STREAMING_ENABLE_DELTA_JOIN;
----
false
```

Reply:
👍 Haven't found this usage before, let me give it a try!